### PR TITLE
Announce DataHub's participation in Hacktoberfest

### PR DESCRIPTION
**Pull request by nagarjunakanamarlapudi**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,10 @@
 > - We've released v0.5.0-beta. You can find release notes [here](https://github.com/linkedin/datahub/releases/tag/v0.5.0-beta)
 > - We're on [Slack](docs/slack.md) now! Ask questions and keep up with the latest announcements.
 
+✨ DataHub participates in Hacktoberfest. Refer https://hacktoberfest.digitalocean.com for more details.
```
Comment (mars-lan) [inline README.md:32]: Move this to top. Also indent the first line like the others

File: README.md [hunk 2]
```diff
@@ -29,6 +29,10 @@
...
+> - Support open source and pick a limited edition T-shirt or plant a tree.
+> - To earn your Hacktoberfest tee or tree reward, you must register and make four valid pull requests (PRs) between October 1-31 (in any time zone)
```
Comment (mars-lan) [inline README.md:34]: Also give more colors on what "valid PR" means.

File: README.md [hunk 3]
```diff
@@ -19,6 +19,10 @@
 
 ![DataHub](docs/imgs/datahub-logo.png)
 
+> ✨ DataHub participates in Hacktoberfest. Refer https://hacktoberfest.digitalocean.com for more details.
```
Comment (mars-lan) [inline README.md:22]: Use this emoticon instead? https://emojipedia.org/t-shirt/

File: README.md (hunk 2 above)
Comment (nagarjunakanamarlapudi) [reply README.md]: https://hacktoberfest.digitalocean.com/details#quality 

This contains what a bad PR is. Let me highlight that instead inline with hacktoberfest 😄

File: README.md (hunk 3 above)
Comment (nagarjunakanamarlapudi) [reply README.md:22]: And https://emojipedia.org/evergreen-tree/ 

( pick a limited edition T-shirt or plant a tree )

Comment (mars-lan) [reply README.md:22]: SGTM

File: README.md (hunk 1 above)
Comment (nagarjunakanamarlapudi) [reply README.md]: Done

**Pull request merged**